source/localizable/index.jp.html.erb

```diff
@@ -69,21 +69,21 @@
 
     <div data-aos="fade-down" class="py-4 px-20 flex">
       <div class="w-1/3 p-8">
-        <img class="block mb-4" src="images/painpoint-1.png" alt="Hard to Focus">
+        <%= image_tag 'images/painpoint-1.png', :class => 'block mb-4', :alt => "Hard to Focus" %>
```

Comment:
this is the old way of writing, better would be to follow `class: 'block mb-4', alt: "Hard to Focus"`

Reply:
Thank you